### DISCUSSION TITLE
Finalizer SWGEmu server

### DISCUSSION THREAD

**Discussion by MrKraken** [pwillworth/galaxyharvester#39] [General]
@pwillworth I was asked if I had another way of contacting you other than the GH email. This is all I can think of. SWGEmu team have submitted new galaxy for review "The Finalizer" goes live today at 16:00 UTC and it would be groovy if the galaxy was approved and active at launch on GH.

Help us Ios-bi Wan Ken-code-bi, you're our only hope

**Comment by pwillworth** [comment]
Wow, thanks for the heads up - I did not get the normal notification on that submission for some reason.  It is now active.

**Reply by MrKraken** [reply]
Winner winner chicken dinner! No worries :)

Hopefully will free up some more time for SWG/GH again in the new year
